packages/lavapack/src/runtime.js

```diff
@@ -11142,6 +11142,11 @@ module.exports = {
   return module.exports
 })()
 
+    const SNOW = globalRef.SNOW;
+    if (!SNOW && includeSnow) {
+      throw 'LavaMoat - Snow is expected to exist but it does not';
```

Comment:
```suggestion
    throw new Error('LavaMoat - Snow is expected to exist but it does not');
```
?
Or if there's a motivation behind throwing a string, an explanatory comment would be great